### PR TITLE
Update rubygem-foreman_remote_execution to 1.6.1

### PR DESCRIPTION
**Pull request by adamruzicka**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by adamruzicka** [comment]
Guess I'll have to rebase this when https://github.com/theforeman/foreman-packaging/pull/2858 gets in

**Comment by evgeni** [comment]
Yepp, please rebase :)

**Comment by adamruzicka** [comment]
Rebased

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-Version:    1.5.5
+Version:    1.6.1
 Release:    1%{?foremandist}%{?dist}
 Group:      Applications/System
 License:    GPLv3
 URL:        https://github.com/theforeman/foreman_remote_execution
 Source0:    https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires:   foreman >= 1.17.0
+Requires:   foreman >= 1.20.0
```
Comment (mmoll) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: This is also needed in line 34 (BuildRequires)

**Pull request merged**